.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Please delete the .idea folder from the repo! It can cause issues when others open the code in phpstorm